cli.go

```diff
@@ -299,6 +317,9 @@ Configuration:
 
   callbackport=<string>
       Optional port to to use in OIDC redirect_uri (default: the value set for port).
+
```

Comment:
Let's update the CLI help text name and description to use `skip_browser`.

Reply:
Whoops! Updated help text.

I fixed whitespace on the other two options here since I am mucking around in the help anyway.

Reply:
Ty!